source/content/guides/composer-convert.md

```diff
@@ -167,36 +167,48 @@ What makes your site code unique is your selection of contributed modules and th
 
-This will list each module followed by the version of that module that is installed.
+We have a script that identifies those modules and puts them in your composer.json file, but the script has limitations:
 
-You can add these modules to your new codebase using Composer by running the following for each module in the `$site-composer` directory. If you use the [Ludwig module](https://www.drupal.org/project/ludwig) do not add it since Composer will take over:
+1. This script DOES NOT resolve your composer.json version problems.
 
-```bash{promptUser:user}
-composer require drupal/MODULE_NAME:^VERSION
-```
+   If there's a version conflict when you `composer install` you will need to resolve them yourself. The first time may be a little painful but going forward, you shouldn't have to do this every time you upgrade. This should be a one-and-done task.
 
-Where `MODULE_NAME` is the machine name of the module in question, and `VERSION` is the version of that module your site is currently using. Composer may pull in a newer version than what you specify, depending upon what versions are available. You can read more about the caret (`^`) [in the documentation for Composer](https://getcomposer.org/doc/articles/versions.md#caret-version-range-) itself.
+1. This script DOES NOT Check for D9 compatibility.
 
-If you get the following error:
+   The Drupal module that does that is called ("upgrade status")[https://drupal.org/project/upgrade_status].
 
-```none
-[InvalidArgumentException]
-Could not find a version of package drupal/MODULE_NAME matching your minimum-stability (stable). Require it with an explicit version constraint allowing its desired stability.
-```
+1. This script DOES NOT Resolve module stability.
+
+   If you get the following error:
+
+   ```none
+   [InvalidArgumentException]
+   Could not find a version of package drupal/MODULE_NAME matching your minimum-stability (stable). Require it with an explicit version constraint allowing its desired stability.
+   ```
+
+   It means that one of the modules you are using (or its dependencies) are not stable. If there is not a stable version you can switch to, you may need to adjust the `minimum-stability` setting of `composer.json` to a more relaxed value, such as `beta`, `alpha`, or even `dev`. You can read more about `minimum-stability` [in the documentation for Composer](https://getcomposer.org/doc/04-schema.md#minimum-stability) itself.
 
-It means that one of the modules you are using (or its dependencies) are not stable. If there is not a stable version you can switch to, you may need to adjust the `minimum-stability` setting of `composer.json` to a more relaxed value, such as `beta`, `alpha`, or even `dev`. You can read more about `minimum-stability` [in the documentation for Composer](https://getcomposer.org/doc/04-schema.md#minimum-stability) itself.
+To run the module migration script, cd to the root directory of your repository and run `bin/migrateModules.php`.
```

Comment:
FOR REFERENCE: https://github.com/pantheon-upstreams/drupal-project/pull/8/files

Reply:
In order for this to be true, my pull request to the other repo must be accepted. That's a completely different argument with @greg-1-anderson that we haven't had yet and the disclaimer that prints at the beginning of the file probably needs a "yes, I know the risks" prompt before it does anything to the composer file.

Reply:
Also, mean "argument" without a negative context. He's a very effective guardrail.

Reply:
No concerns on the wording, but I'd rather not package the script in pantheon-systems/drupal-project. My preference would be to publish an mvp here without a script, and then as a quick follow-on make a new PR that added some scripting, pulling either from a simple gist or perhaps we could register a project in packagist.

Reply:
I've commented the script out for now